Sources/Containerization/SandboxContext/SandboxContext.proto

```diff
@@ -239,6 +241,19 @@ message ConfigureDnsRequest {
 
 message ConfigureDnsResponse {}
 
+message ConfigureHostsRequest {
```

Comment:
do we want a specialized API for this, or perhaps an API to install an arbitrary file?

Reply:
I know we talked on this, but I'm fine with this approach for now after thinking more. I think once the `copy` rpc is a thing we can special purpose that if the thing being copied is a single file and skip the tarball part and just transfer the bytes for the single file directly. We can just remove this rpc if that does come to fruition.